mod_auth/forms.py

```diff
@@ -7,13 +7,24 @@
 
 
 def unique_username(form, field):
-    # Check if a user already exists with this name
+    	"""
```

Comment:
Im not really sure why this alignment error is happening. I gave it exactly one tab space